sim/Makefile

```
@@ -250,6 +250,81 @@ lint:
+	$(RTL_DIR)/periph/interrupt_controller.sv \
+	$(RTL_DIR)/periph/dma_engine.sv \
+	$(RTL_DIR)/soc/axi4_crossbar.sv \
+	$(RTL_DIR)/soc/axi_lite_interconnect.sv \
+	$(RTL_DIR)/soc/sram_controller.sv \
+	$(RTL_DIR)/soc/boot_rom.sv \
+	$(RTL_DIR)/soc/axi4_to_axilite.sv \
+	$(RTL_DIR)/soc/axilite_to_axi4.sv \
+	$(RTL_DIR)/soc/soc_top.sv
+
+.PHONY: lint_soc
+lint_soc:
```

checkmake (0.3.2)

[warning] Target body for "lint_soc" exceeds allowed length of 5 lines (13).

(maxbodylength)